bundles/net.tourbook/src/net/tourbook/data/TourData.java

```diff
@@ -1685,8 +1668,23 @@
    @Transient
    public boolean[]     visiblePoints_ForSurfing;
 
+   /**
+    * An array containing the start time of each pause (in milliseconds)
+    * A timer pause is a device event triggered by the user.
+    */
+   @XmlElementWrapper(name = "PausedTime_Starts")
+   @XmlElement(name = "PausedTime_Start")
+   @Transient
+   public long[]           pausedTime_Start;
 
-// SET_FORMATTING_ON
```

Comment:
Code formatting should be turned on after all fields are defined, otherwise the following code is not formatted

Reply:
Eclipse removed it without asking me. I just put it back.

Reply:
When refactoring code then Eclipse can remove comment lines which are above the refactored code, this is my experience.